docs/code_of_conduct.md

```diff
@@ -0,0 +1,73 @@
+ * Offensive language directed at individuals or groups of people
+ * Bullying (verbal, physical, social, or cyber)
+ * Interfering with someone's ability to contribute, like with excessive nitpicking
+ * Continued one-on-one communication after a party has requested it cease
+ * Stalking online or offline
+ * Doxing or unauthorized publication of private information or communication
+ * Unwelcomed sexual attention
+ * Inappropriate visual displays such as sexually-oriented or offensive  photography, cartoons, drawings, or gestures
+ * Retaliation for reporting or threatening to report harassment
+
+Additionally, spam and other content which disrupts or prevents LDK contributors from working on LDK is not acceptable.
+
+## The Code of Conduct Team
+
+A small team of LDK contributors has volunteered to enforce the LDK Code of Conduct. If you feel like a community member has engaged in inappropriate behavior, please don't hesitate to contact one of the following LDK contributors via email or on Discord:
+ * Matt Corallo - ldkcocpoc on mattcorallo.com
+ * Val Wallace - vwallace on protonmail.com
+ * Devrandom - devrandom99 on proton.me
+
+## The Code of Conduct Team’s Responsibilities
+
+Team members are tasked with responding to reports within 24 hours. They will review each incident and determine, to the best of their ability:
+ * Does the event constitute a Code of Conduct violation?
+ * Is the behavior on our list of inappropriate behavior? Is it borderline inappropriate?
+ * Did the event occur in a space within our Code of Conduct's scope?
+   * If the incident occurred outside community forums and the individual is seen as a project representative or identifies as an LDK contributor, the incident may be in scope.
+   * Additionally, an incident may be in scope if a community member's ability to contribute to LDK is impacted.
+ * Did this incident occur in a private conversation or in a public space?
+ * Is the situation isolated or ongoing?
+ * How is the reported person's behavior negatively impacting others?
+ * Does the incident impact the ability of individuals to freely contribute to LDK?
+ * Does this incident include sexual harassment?
+ * Does this pose a safety risk or severely negatively impact someone's mental health?
+ * Is there a risk of this behavior being repeated?
+ * Does the reported person understand why their behavior was inappropriate?
+
+If a report is insufficiently detailed or involves multiple parties, the Code of Conduct Team may seek additional information from witnesses or the accused. Neither party should contact the other to discuss the incident. Likewise, the team will do its best not to disclose who reported a given incident, either to the accused or generally, though we recognize that circumstantial disclosures to the accused might be unavoidable.
```

Comment:
I don't know much familiarity people have with the notion of "due process" in vigor in most of respectable jurisdictions, and the rights of the defense -- Here we should give far more prerogatives to the "accused" for a fair trial, otherwise we open the door to pure arbitrary. Being an online community doesn't exempt us to respect the defense principles recognized by numerous international treatises and constitutions all around the world.

Among the rights of the defense:
- right to an exact qualification of the torts and the detailed facts
- right to produce a contradictory report with reasonable time offered on all the facts and qualified torts
- right to request a third-party from the Bitcoin community to articulate and defense the case
- right to silence or not to testify against oneself
- right to be communicated all the pieces and documents of the case
- right to appeal the case
- right to ask recusation of a CoC team member if the member is involved in the case
- right to presumption of innocence
- right to proportionality between the tort and the sanction
 
The default delay of 1 week sounds very far from reasonable to produce a grounded defense for the accused and in fact no delay should be given -- It should be function of the respect of the procedural steps guaranteeing the fairness of the disciplinary process.